protos/camera_server/camera_server.proto

```diff
@@ -245,8 +245,9 @@ message Information {
     uint32 horizontal_resolution_px = 7; // Horizontal image resolution in pixels
     uint32 vertical_resolution_px = 8; // Vertical image resolution in pixels
     uint32 lens_id = 9; // Lens ID
-    uint32 definition_file_version = 10; // Camera definition file version (iteration)
-    string definition_file_uri = 11; // Camera definition URI (http or mavlink ftp)
+    uint32 flags = 10; // Bitmap of camera capability flags
```

Comment:
Could this be an enum, instead of a bitmap? It feels like it would make the API clearer. I guess the possible values are defined in MAVLink, right?

Reply:
Yes, I agree, and I was thinking about this too. The problem is that I haven't found a good way to specify flags kind of enum in proto files. The best way I know is to create an enum and then replace the `flags` field type from `uint32` to `Repeated` and parse each flag separately, but it may get very messy. Is there a consistent way do define such fields in MAVSDK-Proto?

Reply:
Would something like [this](https://github.com/mavlink/MAVSDK-Proto/blob/736bc2af1b6eae5c255b5b68014597527bd78c8c/protos/winch/winch.proto#L77-L101) do it?

Reply:
Looks reasonable, thanks for a nice code example. I added the same interface in the following commit

Reply:
Thanks!